src/PanelTraits/Filters.php

```diff
@@ -215,13 +215,22 @@ public function __construct($options, $values, $filter_logic)
 
         $this->values = $values;
         $this->options = $options;
-        $this->view = 'crud::filters.'.$this->type;
+        $this->setView();
 
         if (\Request::has($this->name)) {
             $this->currentValue = \Request::input($this->name);
         }
     }
 
+    public function setView()
+    {
+        if (isset($this->options['view'])) {
+            $this->view = $this->options['view'] . '.' . $this->type;
```

Comment:
Should this not be the exact view without `$this->type` suffix?
Seems like bad naming in the current case... 🤔